programs/ssl/ssl_server2.c

```diff
@@ -1439,7 +1439,7 @@ int dummy_ticket_parse(void *p_ticket, mbedtls_ssl_session *session,
         case 6:
             session->start = mbedtls_time(NULL);
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3)
-            session->ticket_age_add -= 1000;
+            session->ticket_age_add -= 2000;
```

Comment:
This really looks like a statement that should have an explanatory comment

Reply:
Thanks - much better.

Nit: `large` should be `larger` (it's comparative, with `than` following).

Is the following correct? And saying "should be changed when #6788 merged" - changed how? Back to 1000?

```
            /* For this negative test, we need to workaround anti-replay, which
             * adds a 1000ms tolerance to the ticket check. So we reduce the
             * ticket age by 2000ms, which should be enough to trigger the
             * failure we want.
             * Note: previously tried 1000ms, but that's not enough when real
             * age > 1ms.
             * TODO: This needs to be changed (XXX to what?) when PR #6788
             * (which fixes anti-replay failure) is merged.
             */
```